src/Altinn.Apps/AppTemplates/AspNet/App/logic/App.cs

```diff
@@ -45,14 +51,10 @@ public class App : AppBase, IAltinnApp
             IProfile profileService,
             IRegister registerService,
             IPrefill prefillService,
-            IInstance instanceService) : base(
-                appResourcesService,
-                logger,
-                dataService,
-                processService,
-                pdfService,
-                prefillService,
-                instanceService)
+            IInstance instanceService,
+            IOptions<GeneralSettings> settings,
+            IText textService,
+            IHttpContextAccessor httpContextAccessor) : base(appResourcesService, logger, dataService, processService, pdfService, prefillService, instanceService, registerService, settings, profileService, textService, httpContextAccessor)
         {
             _logger = logger;
             _validationHandler = new ValidationHandler();
```

Comment:
Now that HttpContextAccessor is injected into this class, I would like to see that it is sent into ValidationHandler as well. currently, it is an optional parameter with default null which is a bit confusing.

Reply:
@acn-sbuad  good point. Added it